frontend/document_collaboration_system/src/messageDialog.js

```diff
@@ -0,0 +1,121 @@
+import React, { useCallback, useContext } from "react";
+import { AppContext } from "./index";
+import Dialog from "@material-ui/core/Dialog";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import FormControl from "@material-ui/core/FormControl";
+import InputLabel from "@material-ui/core/InputLabel";
+import Select from "@material-ui/core/Select";
+import MenuItem from "@material-ui/core/MenuItem";
+import { TextField } from "@material-ui/core";
+import DialogActions from "@material-ui/core/DialogActions";
+import Button from "@material-ui/core/Button";
+import { api } from "./service";
+
+export default function SendMessageDialog({
+  openDialog,
+  setOpen,
+  messageMode,
+  users,
+}) {
+  const [message, setMessage] = React.useState("");
```

Comment:
why don't you destructed useState on the file top? and maybe you don't have to import React in 17 version